packages/simple-hub/src/logger.ts

```diff
@@ -1,10 +1,12 @@
 import pino from 'pino';
+import {LOG_DESTINATION} from './constants';
+const LOG_TO_CONSOLE = LOG_DESTINATION === 'console';
+const LOG_TO_FILE = !LOG_TO_CONSOLE;
 
-const _logger = pino();
+const name = 'simple-hub';
 
-export const log = {
-  debug: _logger.debug.bind(_logger),
-  info: _logger.info.bind(_logger),
-  error: _logger.error.bind(_logger),
-  fatal: _logger.fatal.bind(_logger)
-};
+const destination = LOG_TO_FILE ? pino.destination(LOG_DESTINATION) : undefined;
+const prettyPrint = LOG_TO_CONSOLE ? {translateTime: true, color: true} : false;
+
+const opts = {name, prettyPrint};
+export const log = destination ? pino(opts, destination) : pino(opts);
```

Comment:
Surprised this works. I was seeing some issues with pino + observables and had to add `this` binding to get around those.

Reply:
You mean `pino().info` was not working from the context of an observable?

Reply:
I think that `log.info` was not working where log was defined outside of observable.